go/vt/discovery/tablet_health.go

```diff
@@ -32,13 +32,16 @@ import (
 // TabletHealth represents simple tablet health data that is returned to users of healthcheck.
 // No synchronization is required because we always return a copy.
 type TabletHealth struct {
-	Conn                 queryservice.QueryService
+	Key                  string
+	Conn                 queryservice.QueryService `json:"-"`
 	Tablet               *topodata.Tablet
+	Name                 string
 	Target               *query.Target
+	Up                   bool
+	Serving              bool
+	PrimaryTermStartTime int64 `json:"TabletExternallyReparentedTimestamp"`
 	Stats                *query.RealtimeStats
-	PrimaryTermStartTime int64
 	LastError            error
```

Comment:
In order to be backward compatible, I had to add/rename/reorder fields.

The following fields are added: `Key`, `Name`, `Up`. This is done to ensure the JSON output matches the one we produce in previous releases.

The following field is renamed: `PrimaryTermStartTime` into `TabletExternallyReparentedTimestamp` (only in the JSON). I am considering adding a duplicated struct field of `PrimaryTermStartTime` that will use the same JSON name. That way people can switch to the `PrimaryTermStartTime` JSON field in v14, and we will be able to drop the `TabletExternallyReparentedTimestamp` field in v15. Let me know what you think.

Reply:
Where are we using Key/Name?
No need to add `Up` here since we are hardcoding it wherever needed.
For `PrimaryTermStartTime` instead of duplicating the struct field we should be able to write a custom formatter (or something like that) which produces two fields in the JSON from one field in the struct.

Reply:
oh, and `Serving` should be inside `Stats` so no need to add another field for that either.

Reply:
@frouioui pointed out that `Serving` is not a new field. The diff was hard to read 🙄